examples/dem/3d-bunny-drill/bunny-drill-loading.prm

```diff
@@ -91,14 +91,14 @@ end
 #---------------------------------------------------
 
 subsection insertion info
-  set insertion method                               = volume
-  set inserted number of particles at each time step = 10000
-  set insertion frequency                            = 100000
-  set insertion box points coordinates               = 0.1, -0.07, -0.07 : 0.24, 0.07, 0.07
-  set insertion distance threshold                   = 1.10
-  set insertion maximum offset                       = 0.10
+  set insertion method                               = plane
+  set inserted number of particles at each time step = 200
+  set insertion frequency                            = 8000
+  set insertion plane point                          = 0.025, 0, 0
+  set insertion plane normal vector                  = -1, 0, 0
+  set insertion maximum offset                       = 0.00
   set insertion prn seed                             = 19
-  set insertion direction sequence                   = 0, 2, 1
+  set initial velocity                               = -0.1, 0.0, 0.0
```

Comment:
You could load the particles faster with an initial velocity in Y.

Reply:
It is already pretty quick with the insertion plane

Reply:
Tehy already have an initial velocity. The main axis of the cylinder is X